front/src/modules/timetableItem/components/Timetable/PacedTrain/hooks/useOccurrences.ts

```diff
@@ -73,7 +73,7 @@ const useOccurrences = (
         disabled: correspondingException?.disabled,
         // In the model, we can currently have a null category value so we need to handle this case
         category: correspondingException?.rolling_stock_category
-          ? correspondingException.rolling_stock_category.value
+          ? correspondingException?.rolling_stock_category?.value
```

Comment:
Nit: I think these `?` aren't required since we check that it's non-null in the condition above.